pkg/option/config.go

```diff
@@ -1011,6 +1012,34 @@ var HelpFlagSections = []FlagsSection{
 			IpvlanMasterDevice,
 		},
 	},
+	{
+		Name: "KubeProxy free flags",
+		Flags: []string{
+			KubeProxyReplacement,
+			EnableNodePort,
+			EnableHostReachableServices,
+			EnableExternalIPs,
+			HostReachableServicesProtos,
+			NodePortMode,
+			EnableAutoDirectRoutingName,
```

Comment:
I don't think  AutoDirectRouting is related to kube-proxy free. Its existing listing under Route flags is fine.

Reply:
Yes I moved it back.